client_lib/pulp/client/commands/repo/cudl.py

```diff
@@ -522,16 +522,22 @@ def _default_summary_view(repo_list, prompt):
 
     if isinstance(repo_list, dict) and repo_list != {}:
         id_value = repo_list['id'] + ' '
-        name_value = repo_list['display_name']
```

Comment:
Actually, what do you think about making a `name_value` an empty string and leave everything else as is to preserve the output syntax, just in case someone parses it?
Something like
`name_value = repo_list['display_name'] if repo_list['display_name'] is not None else ''`